package.json

```diff
@@ -2,19 +2,21 @@
   "name": "@netlify/zip-it-and-ship-it",
   "version": "4.23.2",
   "description": "Zip it and ship it",
-  "main": "./src/main.js",
+  "main": "./dist/main.js",
   "bin": {
-    "zip-it-and-ship-it": "src/bin.js"
+    "zip-it-and-ship-it": "dist/bin.js"
   },
   "files": [
-    "src/**/*.js"
+    "dist"
   ],
   "scripts": {
     "prepublishOnly": "npm ci && npm test",
+    "prepack": "npm run build",
+    "build": "tsc && chmod +x dist/bin.js",
```

Comment:
Is there a cross-OS way to ensure the right permissions for the binary?

Reply:
Adding that permission is only required for the `execa` call in `tests/bin.ts`. Now that I think of it, we should be able to remove the `chmod +x` call altogether if we run `execa("node dist/bin.js")` instead of `execa("dist/bin.js")`.